MobileClient/SignupScreen.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { StyleSheet, Text, View, Image, TextInput, TouchableOpacity } from "react-native";
+import Feather from 'react-native-vector-icons/Feather';
+
+export default function SignupScreen() {
+    Feather.loadFont();
+
+    const [entryP, setEntryP] = useState(true); //state for Password
+    const [iconP, setIconP] = useState("eye-off");
+
+    const [entryC, setEntryC] = useState(true); //state for Confirm Password
+    const [iconC, setIconC] = useState("eye-off");
+
+    function handlePress_Pass() {
+        setIconP(entryP ? "eye" : "eye-off");
+        setEntryP(!entryP);
+    }
+
+    function handlePress_Confirm() {
```

Comment:
Try to stick with camelCase only rather than combining camelCase and snake_case

Reply:
Okay